packages/gatsby-theme-aio/src/components/Tabs/index.js

```diff
@@ -24,52 +27,379 @@ const animateIndicator = (indicator, animate) => {
+              const ref = createRef();
+              tabs.push(ref);
+              const isSelected = selectedIndex.tab === index;
+              return (
+                <Item
+                  className={"tabItem"}
+                  key={`tabItem_${index}`}
+                  tabIndex={0}
+                  ref={ref}
+                  isSelected={isSelected}
+                  label={<b>{data['heading']}</b>}
+                  icon={data['image']}
+                  onClick={() => {
+                    setSelectedIndex({
+                      tab: index,
+                    });
+                    positionSelectedTabIndicator(index);
+                  }}
+                  css={css`
+                    text-align: left;
+                    white-space: normal;
+                    display: flex !important;
+                    height: auto !important;
+                    padding: var(--spectrum-global-dimension-size-200) var(--spectrum-global-dimension-size-200) var(--spectrum-global-dimension-size-250) !important;
+                    margin-bottom: var(--spectrum-global-dimension-size-350) !important;
+                    width:${layoutColumns(3)} !important;
+                    .spectrum-Tabs-itemLabel{
+                      font-size:var(--spectrum-global-dimension-size-185);
+                      font-weight:bold;
+                      margin-left: var(--spectrum-global-dimension-size-200);
+                      line-height: initial;
+                      margin-top:var(--spectrum-global-dimension-size-100);
+                    }
+
+                    .spectrum-Icon{
+                      width: 40px;
+                      height: 40px;
+                    }
+
+                    @media only screen and (max-width: 480px) {
+                      max-width: ${layoutColumns(2.5)} !important;
+                      justify-content: left;
+                      margin-bottom: var(--spectrum-global-dimension-size-150) !important;
+
+                      .spectrum-Tabs-itemLabel{
+                        margin-top:var(--spectrum-global-dimension-size-50) !important;
+                      }
+                    }
+
+                    @media only screen and (device-width: ${TABLET_SCREEN_WIDTH}) {
+                      max-width: ${layoutColumns(2)} !important;
+                    }
+                    @media only screen and (device-width: ${MOBILE_SCREEN_WIDTH}) {
+                      max-width: ${layoutColumns(2)} !important;
+                    }
+                    &.is-disabled {
+                      pointer-events:none;
+                    }
+                    &:hover, &.is-selected {
+                        background-color: var(--spectrum-alias-background-color-gray-50, var(--spectrum-global-color-gray-50, var(--spectrum-semantic-gray-50-color-background)));
+                        color: var(--spectrum-global-color-gray-900);
+                        box-shadow: 0 3px 16px #46303029;
```

Comment:
Is this a custom color or is it available as Spectrum CSS variable ?

Reply:
This Spectrum CSS variable.